MeshLib/IO/VtkIO/PVDFile.h

```diff
@@ -33,6 +33,8 @@ class PVDFile
     //! Add a VTU file to this PVD file.
     void addVTUFile(std::string const& vtu_fname, double timestep);
 
+    std::string const& getName() const;
+
 private:
     std::string const _pvd_filename;
```

Comment:
I'd just move it `const` as it is to public section and don't add the extra `getName()`.
```suggestion
public:
    std::string const pvd_filename;
```

Reply:
Yes, I know less code contains fewer errors. But sorry, I'm old school object oriented programmer. I'd like to keep it that way.

Reply:
Changed to @endJunction suggested version. It's not only less code it is even less assembly code and thus performs better. Since the class is very simple we can do without encapsulation.